etc/orthography.tsv

```diff
@@ -0,0 +1,88 @@
+Grapheme	IPA	EXAMPLES
+^	NULL	0
+$	NULL	0
+-	NULL	wai-oot;i-thlung-a;i-thlung-a;i-hling-a;i-hling-a
```

Comment:
I know you mentioned these are not necessarily morpheme boundaries, sometimes just a syllable one, but perhaps it is worth to map it as `-/` (using the slash notation and nothing to the right). Thoughts, @LinguList ?

Reply:
Yes, `-` is more a 'pronunciation unit' as it is a morpheme marker or syllable marker. Happy to learn how this could be incorporated.